spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -5986,32 +6021,33 @@ and/or a `WebDriver` bean. Here is an example that uses HtmlUnit:
 	}
 ----
 
-NOTE: By default Spring Boot will put `WebDriver` beans in a special "`scope`" to ensure
-that the driver is quit after each test, and that a new instance is injected. If you don't
-want this behavior you can add `@Scope("singleton")` to your `WebDriver` `@Bean`
+NOTE: By default, Spring Boot puts `WebDriver` beans in a special "`scope`" to ensure
+that the driver is quit after each test and that a new instance is injected. If you do not
+want this behavior, you can add `@Scope("singleton")` to your `WebDriver` `@Bean`
 definition.
 
-A list of the auto-configuration that is enabled by `@WebMvcTest` can be
+A list of the auto-configuration settings that are enabled by `@WebMvcTest` can be
 <<appendix-test-auto-configuration#test-auto-configuration,found in the appendix>>.
 
 
 
 [[boot-features-testing-spring-boot-applications-testing-autoconfigured-webflux-tests]]
-==== Auto-configured Spring WebFlux tests
-To test Spring WebFlux controllers are working as expected you can use the `@WebFluxTest`
-annotation. `@WebFluxTest` will auto-configure the Spring WebFlux infrastructure and limit
-scanned beans to `@Controller`, `@ControllerAdvice`, `@JsonComponent`,and
-`WebFluxConfigurer`. Regular `@Component` beans will not be scanned when using this
-annotation.
+==== Auto-configured Spring WebFlux Tests
+To test that Spring WebFlux controllers are working as expected, you can use the `@WebFluxTest`
+annotation. `@WebFluxTest` auto-configures the Spring WebFlux infrastructure and limits
+scanned beans to `@Controller`, `@ControllerAdvice`, `@JsonComponent`, and
+`WebFluxConfigurer`. Regular `@Component` beans are not scanned when the `@WebFluxTest`
+annotation is used.
 
-Often `@WebFluxTest` will be limited to a single controller and used in combination with
-`@MockBean` to provide mock implementations for required collaborators.
+Often, `@WebFluxTest` is limited to a single controller and used in combination with the
+`@MockBean` annotation to provide mock implementations for required collaborators.
 
 `@WebFluxTest` also auto-configures `WebTestClient`, which offers a powerful way to quickly
 test WebFlux controllers without needing to start a full HTTP server.
 
 TIP: You can also auto-configure `WebTestClient` in a non-`@WebFluxTest`
-(e.g. `SpringBootTest`) by annotating it with `@AutoConfigureWebTestClient`.
+(such as `SpringBootTest`) by annotating it with `@AutoConfigureWebTestClient`. The
```

Comment:
`SpringBootTest` -> `@SpringBootTest`?

Reply:
OK. I don't know Spring well enough to recognize more than the most common annotations (one of many reasons I love a good review). Thanks.